docs/enhancements/arbiter-clusters.md

```diff
@@ -0,0 +1,134 @@
+---
+title: arbiter-clusters
+authors:
+  - "@giladravid16"
+creation-date: 2025-03-13
+last-updated: 2025-03-13
+---
+
+# Support 2 Node + 1 Arbiter Node HA Cluster
+
+## Summary
+
+This enhancement describes the ability to install TNA (Two Nodes + Arbiter)
+clusters - OpenShift clusters with control planes that consists of at minimum
+2 normal-sized nodes, and at least 1 node that can be less powerful than the
+recommended node size.
+Arbiter nodes will only be running critical components for maintaining 
+HA (e.g etcd) to allow the arbiter node size to be as small and as low cost as
+possible within reason.
+
+## Motivation
+
+Customers at the edge are requiring a more economical solution for HA deployments
+at the edge. They can support running 2 node clusters for redundancy but would like 
+the option to deploy a lower cost node as an arbiter to supply the 3 nodes for etcd quorum.
+
+### Goals
+
+- Allow for a TNA cluster to be installed.
+
+### Non-Goals
+
+- Moving from 2 + 1 to a conventional 3 node cluster.
+
+## Proposal
+
+This enhancement will include the following:
+- In the API we will:
+  - Add a new host_role for arbiter.
+  - Add a new high_availability_mode for TNA.
+  - Deprecate the existing parameters and create new ones for hyperthreading
+    and disk encryption (they will be maps of host_role to bool).
+  - Add a new feature_support_level_id for TNA.
+- We will update the clusters' validations and transitions for TNA clusters.
```

Comment:
can you add more details about the validations that will be affected by this change?